src/hooks/useActions.js

```diff
@@ -80,15 +80,20 @@ export default function useActions(onDone) {
   )
 
   const stakeToProposal = useCallback(
-    (proposalId, amount) => {
-      sendIntent(convictionVotingApp, 'stakeToProposal', [proposalId, amount], {
-        ethers,
-        from: account,
-      })
+    async ({ proposalId, amount }, onDone = noop) => {
+      const intent = await agreementApp.intent(
```

Comment:
here is where you are having the issue, is not the agreement app we stake with the `convictionVotingApp`